website/config.yml

```diff
@@ -4,29 +4,51 @@ title: PRQL
 theme: prql-theme
 
 disableKinds:
-- taxonomy
-- term
+  - taxonomy
+  - term
 
 menu:
   nav:
+    # TODO: Do we need identifier & name & title?
+    # TODO: Do we need url & pageref?
+    # TODO: Can we use the info from pages directly? https://gohugo.io/templates/menu-templates/#menu-entries-from-the-pages-front-matter
     - identifier: Book
       name: Book
       title: Book
       url: "https://prql-lang.org/book/"
       pageref: "https://prql-lang.org/book/"
+      # TODO: Could we automatically do these three lines for external links, across the whole site?
+      post: <sup><i class="bx bx-link-external"></i></sup>
+      params:
+        target: "_blank"
       weight: 1
     - identifier: Playground
       name: Playground
       title: Playground
       url: "https://prql-lang.org/playground/"
       pageref: "https://prql-lang.org/playground/"
       weight: 2
+      post: <sup><i class="bx bx-link-external"></i></sup>
+      params:
+        target: "_blank"
     - identifier: Roadmap
       name: Roadmap
       title: Roadmap
       url: /roadmap/
       pageref: /roadmap/
       weight: 3
+    - identifier: Examples
+      name: Examples
+      title: Examples
+      url: /examples/
+      pageref: /examples/
+      weight: 4
+    - identifier: FAQ
```

Comment:
This could be cleaned-up: `weight`, `pageref` and `title` are unused. 

I don't know why the author of the theme even included them here...